.circleci/config.yml

```diff
@@ -123,10 +124,11 @@ jobs:
             mkdir /logs
             conda activate esmvaltool
             pip install .[test] > /logs/install.txt 2>&1
-            # Run flake8 check with source code (it doesn't work without it)
-            pytest -n 4 -m flake8
             # Update/install Julia dependencies
             esmvaltool install Julia > /logs/install_julia.txt 2>&1
+            # Run flake8 (and mo pytest)
+            # TODO Bouwe, check this pls
+            flake8 -j 4
```

Comment:
@bouweandela am a bit confused here - why are we not actually running the full pytests but only the flake sub-marked? Am always bemused by this bit of Circle. Is this OK what I'm doing here?

Reply:
You cannot lint source code that has been deleted, so we run flake8 to lint the source code and then delete it to make sure we test the installed version of the software because this is how our users will use it.

Could you keep the old line 126 (which tries to explain this) and change the old line 127 so it runs flake8 directly? and remove the new line 129 to 131 please?

Reply:
done! Thanks a lot, bud :beer: